### ISSUE TITLE
`at-rule-no-unknown` false positive for `@view-transition`

### ISSUE THREAD

**Issue by sebdanielsson** [stylelint/stylelint#7750]
### What minimal example or steps are needed to reproduce the bug?

```css
@view-transition {
  navigation: auto;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "at-rule-no-unknown": true
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css"
```

### Which Stylelint-related dependencies are you using?

```json
{
    "stylelint": "^16.6.1",
    "stylelint-config-html": "^1.1.0",
    "stylelint-config-standard": "^36.0.0",
}
```

### What did you expect to happen?

Lint should pass.

### What actually happened?

```
20:3  ✖  Unexpected unknown at-rule "@view-transition"  at-rule-no-unknown
21:5  ✖  Unexpected unknown property "navigation"       property-no-unknown
```

**Comment by Mouvedia** [comment]
ref

https://developer.chrome.com/docs/web-platform/view-transitions#how_to_trigger_2
https://www.w3.org/TR/css-view-transitions-2/#view-transition-rule

**Comment by Mouvedia** [comment]
I've labeled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.
